.github/workflows/main.yml

```diff
@@ -121,26 +94,24 @@ jobs:
 
   msrv:
     name: MSRV
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-24.04
```

Comment:
Pin the version to avoid unexpected pointer changes.

Reply:
What kinds of unexpected pointer changes are you meaning here?

Reply:
I mean GitHub can change what `ubuntu-latest` means, perhaps "ubuntu 26.04" one day.

Previously, some of projects I participants find their CI failed due to this unexpected bump, while using a pinned version give us the freedom to bump on demand and more manageable (rather than affect all the new PRs).

Reply:
This seems like something we'll simply not do if it doesn't stop working, so I'd rather leave it and see if does end up breaking.